frontend/packages/console-dynamic-plugin-sdk/src/extensions/add-actions.ts

```diff
@@ -19,6 +19,11 @@ export type AddAction = ExtensionDeclaration<
     icon?: CodeRef<React.ReactNode>;
     /** Optional access review to control visibility / enablement of the action. */
     accessReview?: AccessReviewResourceAttributes[];
+    /** action to specify add action as a topology context menu item */
+    action?: {
+      name?: string;
+      isGroupable: boolean;
```

Comment:
add a comment here noting.. `isGroupable` is to denote that resources created through this flow are groupable or not